### PR TITLE
Add beta features file

### PR DESCRIPTION
**Pull request by elisallenens**
This is to be used together with the modelchecker for raising an error on any Spatialite database where users are incorrectly making use of columns still in beta status. See ticket nens/threedi-modelchecker#293.

### CODE REVIEW

**Comment by jpprins1** [comment]
Note: There might already be columns, please check with @martijn-siemerink

**Comment by martijn-siemerink** [comment]
@elisallenens  @jpprins1.  Zeker. De velden uit migraties 215 en 216 moeten opgenomen in de Beta features lijst.

File: threedi_schema/beta_features.py [hunk 1]
```diff
@@ -0,0 +1,48 @@
+from .domain import constants, models
+
+"""
+Put features in beta development in these lists to prevent users from using them, in the SQLAlchemy format used in the schema.
+"""
+"""This list contains beta tables, e.g. models.SomeTable"""
+BETA_TABLES = [
```
Comment (caspervdw) [inline threedi_schema/beta_features.py:7]: The table itself is indeed beta, but I think it is easier to put the "beta" flag on the `models.GlobalSetting.vegetation_id`. Then it falls in the `BETA_COLUMNS` category; so if someone has a record in the vegetation table, that's ok, as long as it is not used.

File: threedi_schema/beta_features.py [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+    "columns": [
+        models.SomeTable.some_column,
+        models.SomeTable.some_other_column,
+    ],
+    "values": [
+        constants.SomeConstantsClass.SOMEENUMVALUE,
+        constants.SomeConstantsClass.SOMEOTHERENUMVALUE,
+    ]
+}
+The columns in which the enums are used are included so that the modelchecker doesn't have to
```
Comment (caspervdw) [inline threedi_schema/beta_features.py]: I think it is not necessary to say this.

What is necessary is to describe the result of your example.

**Pull request merged**